sympy/printing/mathml.py

```diff
@@ -1271,6 +1271,25 @@ def _print_Lambda(self, e):
         return x
 
 
+    def _print_tuple(self, e):
+        x = self.dom.createElement('mfenced')
+        for i in e:
+            x.appendChild(self._print(i))
+        return x
+
+
+    def _print_IndexedBase(self, e):
+        return self._print(e.label)
+
+    def _print_Indexed(self, e):
+        x = self.dom.createElement('msub')
+        x.appendChild(self._print(e.base))
+        if len(e.indices) == 1:
+            x.appendChild(self._print(e.indices[0]))
+            return x
+        x.appendChild(self._print(e.indices))
```

Comment:
There should be a return here, right? Or is it that indices[0] is a tuple when there is more than one? (If the tests pass.) Apart from that this looks good and the new tests look good. Maybe one could also add explicit tests for `tuple` and possibly `Indexed` although I understand that they are covered by the current test.

Reply:
added return x, Thanks for pointing that out.